lib/compat/wordpress-6.0/block-patterns-update.php

```diff
@@ -157,6 +157,18 @@ function gutenberg_register_gutenberg_patterns() {
 								<!-- wp:social-link {"url":"#","service":"mail"} /--></ul>
 								<!-- /wp:social-links -->',
 		),
+		'author-biograhy-border'               => array(
+			'title'      => _x( 'Post author biography with avatar and border', 'Block pattern title', 'gutenberg' ),
+			'categories' => array( 'text' ),
+			'blockTypes' => array( 'core/post-author-name', 'core/post-author-biography', 'core/avatar' ),
```

Comment:
It means that we would need all 3 blocks to be promoted to stable to include this new pattern in WordPress 6.0.

Reply:
yes

Reply:
Or did you mean, that they should be moved to stable with this PR?

Reply:
I was thinking we should do it separately as a pre-requisite for this PR.